src/server/apollo.ts

```diff
@@ -27,6 +27,27 @@ import { EventEmitter } from 'events';
 import { initAccountDeletionCompleteEvents } from '../aws/eventTypes';
 import { typeDefs } from './typeDefs';
 import { resolvers } from '../resolvers';
+import { Knex } from 'knex';
+
+/**
```

Comment:
other options considered is checking the resolver chain's `info` object on if the parent's Operation type is mutation..but this fix is more concrete at one place when the request comes in, and we don't have to amend context on every resolver